File: Example/OCSlimProject_FitnesseTests/OCSlimFitnesseTestReportReaderTests.m
#import <XCTest/XCTest.h>
#import "OCSlimFitnesseTestReportReader.h"
#import "OCSlimProjectTestDataManager.h"
#import "OCSlimFitnesseTestReportReader.h"

@interface OCSlimFitnesseTestReportReaderTests : XCTestCase

@property (nonatomic, strong) OCSlimFitnesseTestReportFileReaderTests* reader;
@end

@implementation OCSlimFitnesseTestReportReaderTests

- (void)setUp {
    [super setUp];
    
    self.reader = [[OCSlimFitnesseTestReportFileReaderTests alloc] init];
    
}

- (void)tearDown {
    
    self.reader = nil;
    
    [super tearDown];
}

- (void) testReaderDataEqualsFitnesseReportFileData {

    NSData *data = [OCSlimProjectTestDataManager fitnesseTestReportData];
    
    XCTAssertEqual(data, [self.reader read]);
}

@end

File: Example/OCSlimProject_FitnesseTests/OCSlimFitnesseTestReportReaderTests.swift
import XCTest

class OCSlimFitnesseTestReportReaderTests: XCTestCase {

    let reader = OCSlimFitnesseTestReportFileReaderTests()
    
    func testReaderDataEqualsFitnesseReportFileData() {
        
        let data = OCSlimProjectTestDataManager().fitnesseTestReportData
        
        XCTAssertEqual(data, reader.read())
    }

}
